tests/test-cases/Group0-Bugs/4817.robot

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 4817
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${false}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+Check kernel vsyscall
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -t centos:6 bash -c date
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
Can we check the date too to at least see that it mostly works? Maybe just the day/month/year, don't really care about time accuracy, but would like to see that the command returns something reasonable while we are here.